kodi_addon_checker/check_addon.py

```diff
@@ -137,11 +141,12 @@ def _check_addon_xml(report: Report, addon_path):
     try:
         _addon_file_exists(report, addon_path, r"addon\.xml")
 
-        addon_xml = xml.etree.ElementTree.parse(addon_xml_path)
+        addon_xml = ET.parse(addon_xml_path)
         addon = addon_xml.getroot()
-        report.add(Record(INFORMATION, "Created by %s" % addon.attrib.get("provider-name")))
+        report.add(Record(PROBLEM, "Addon xml not valid, check xml. %s" % relative_path(addon_xml_path)))
```

Comment:
The other one is okay now, but you changed this one now. I don't think you ment to do this.

Reply:
I am really sorry for this. I'll fix it right away.